src/components/TodoList/TodoList.tsx

```diff
@@ -25,184 +56,57 @@ export const TodoList: React.FC = () => {
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">9</td>
-            <td className="is-vcentered"> </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">molestiae perspiciatis ipsa</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">10</td>
-
-            <td className="is-vcentered">
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
-            </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-success">illo est ratione doloremque quia maiores aut</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
+          {todos.map(todo => (
+            <tr
+              data-cy="todo"
+              key={todo.id}
+              className={todo === currentTodo ? 'has-background-info-light' : ''}
+            >
+              <td className="is-vcentered">
+                {todo.id}
+              </td>
+              <td className="is-vcentered">
+                {todo.completed && (
+                  <span
+                    className="icon"
+                    data-cy="iconCompleted"
+                  >
+                    <i
+                      className="fas fa-check"
+                    />
+                  </span>
+                )}
+              </td>
+
+              <td className="is-vcentered is-expanded">
+                <p
+                  className={todo.completed
+                    ? 'has-text-success'
+                    : 'has-text-danger'}
+                >
+                  {todo.title}
+                </p>
+              </td>
+
+              <td className="has-text-right is-vcentered">
+                <button
+                  data-cy="selectButton"
+                  className="button"
+                  type="button"
+                  onClick={() => {
+                    dispatch(actionsWithTodo.set(todo));
+                  }}
```

Comment:
do not save the same value in 2 places (todo[] <=> selectedTodo), save todoId here

Reply:
changed.